cadence/transactions/create_registry.cdc

```diff
@@ -0,0 +1,16 @@
+import "FLIXRegistry"
+
+transaction {
+
+    prepare(signer: AuthAccount) {
```

Comment:
if we could have another TX like this that says create unstable registry where the name of the registry is the path.

Reply:
Do we need an entirely separate TX or could it just be updated like the below

```cadence
import "FLIXRegistry"

transaction(name: String?) {

    prepare(signer: AuthAccount) {
        // Check if the account already has a Registry
        if signer.borrow<&FLIXRegistry.Registry>(from: /storage/stableFlixRegistry) == nil {
            // Create a new Registry resource
            let registry <- FLIXRegistry.createRegistry()

            var path = /storage/flix

            if name != nil {
                path = StoragePath(identifier: path.toString().concat("_").concat(name!))!
            }

            // Save it with restricted access
            signer.save(<-registry, to: path)
            signer.link<&FLIXRegistry.Registry{FLIXRegistry.Queryable, FLIXRegistry.Admin}>(/public/stableFlixRegistryPublic, target: /storage/stableFlixRegistry)
        }
    }
}
```

Reply:
code updated